webui.py

```
@@ -315,6 +336,8 @@ async def run_custom_agent(
         use_vision,
         max_actions_per_step,
         tool_calling_method
+        gbc,
```

Ruff (0.8.2)

SyntaxError: Expected ',', found name

---

Trailing comma missing

(COM812)